kubernetes/ingress/nginx/nginx-ingress-controller.yaml

```diff
@@ -19,7 +38,7 @@ spec:
           args:
              - /nginx-ingress-controller
              - --default-backend-service=default/default-http-backend
-             - --default-ssl-certificate=$(POD_NAMESPACE)/tls-certificate
+             - --default-ssl-certificate=default/okapi-tls
```

Comment:
this sort of makes sense, but why was it working before?  what prompted this change?  the ingress itself is pointing to `okapi-tls`, so i assumed the "default ssl certificate" in this controller was just some throwaway that would get replaced when the ingress object was applied

Reply:
Well it can go either way. What i thought was happening was the same thing that was happening before when trying to hit the cluster from a Client that didn't originate from the host and then getting back the fake Kubernetes cert and the pem chain.